doc/source/getting_started/install.rst

```diff
@@ -30,197 +31,133 @@ Installing pandas
 In Linux/Mac you can run ``which python`` on your terminal and it will tell you which Python installation you're
 using. If it's something like "/usr/bin/python", you're using the Python from the system, which is not recommended.
 
 It is highly recommended to use ``conda``, for quick installation and for package and dependency updates.
-You can find simple installation instructions for pandas in this document: ``installation instructions </getting_started.html>``.
+You can find simple installation instructions for pandas :ref:`in this document <install.miniconda>`.
+
+.. _install.source:
 
 Installing from source
 ~~~~~~~~~~~~~~~~~~~~~~
 
-See the :ref:`contributing guide <contributing>` for complete instructions on building from the git source tree. Further, see :ref:`creating a development environment <contributing_environment>` if you wish to create a pandas development environment.
+See the :ref:`contributing guide <contributing>` for complete instructions on building from the git source tree.
+Further, see :ref:`creating a development environment <contributing_environment>` if you wish to create
+a pandas development environment.
+
+.. _install.dev:
 
 Installing the development version of pandas
 ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
 
-Installing a nightly build is the quickest way to:
+Installing the development version is the quickest way to:
 
 * Try a new feature that will be shipped in the next release (that is, a feature from a pull-request that was recently merged to the main branch).
 * Check whether a bug you encountered has been fixed since the last release.
 
-You can install the nightly build of pandas using the scientific-python-nightly-wheels index from the PyPI registry of anaconda.org with the following command::
+The development version is usually uploaded daily to the scientific-python-nightly-wheels
+index from the PyPI registry of anaconda.org. You can install it by running.
+
+.. code-block:: shell
 
     pip install --pre --extra-index https://pypi.anaconda.org/scientific-python-nightly-wheels/simple pandas
 
-Note that first uninstalling pandas might be required to be able to install nightly builds::
+Note that you might be required to uninstall an existing version of pandas to install the development version.
+
+.. code-block:: shell
 
     pip uninstall pandas -y
 
 Running the test suite
 ----------------------
 
-pandas is equipped with an exhaustive set of unit tests, covering about 97% of
-the code base as of this writing. To run it on your machine to verify that
-everything is working (and that you have all of the dependencies, soft and hard,
-installed), make sure you have `pytest
+pandas is equipped with an exhaustive set of unit tests. The unit tests require `pytest
 <https://docs.pytest.org/en/latest/>`__ >= 7.3.2 and `Hypothesis
-<https://hypothesis.readthedocs.io/en/latest/>`__ >= 6.34.2, then run:
+<https://hypothesis.readthedocs.io/en/latest/>`__ >= 6.34.2 to run. To run the tests from a
```

Comment:
IIRC you need more than this.

I think you need pytest-asyncio and pytest-localserver(??? not sure if this was removed).

Reply:
I realized we have a `pandas[test]` extra that covers all the dependencies so I added it here.